build.sbt

```diff
@@ -188,6 +188,7 @@ lazy val docs = project
     moduleName := "circe-docs",
     name := "Circe docs",
     mdocIn := file("docs/src/main/tut"),
+    evictionErrorLevel := Level.Warn,
```

Comment:
Dealing with circe-optics is ok but untangling circe-generic-extras from the docs is harder. This will get us by for now.